src/content_mapblock.h

```diff
@@ -26,10 +26,27 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 struct MeshMakeData;
 struct MeshCollector;
 
+struct LegacyLightingPair
```

Comment:
hmm, really. it could be used in `convert_lighting` but it doesn’t seem to need such struct